jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/Request.java

```diff
@@ -675,9 +675,10 @@ public default void onFailure(Request request, Throwable failure)
     public interface Content extends org.eclipse.jetty.io.Content.Source
     {
         /**
-         * @return the content type string such as "application/octet-stream" or
-         * "application/json;charset=UTF8", or null if the {@code Content-Type}
-         * header must not be set
+         * @return the value of the {@code Content-Type} header for the request
+         * content, such as {@code application/octet-stream} or {@code application/json},
```

Comment:
Does this no longer return the charset?

Reply:
It was just an example, actually `application/json` does not need a charset since it is implicitly UTF-8.

The method returns the `Content-Type`, which is a mime type plus an optional charset, so nothing has changed there.

I changed the javadocs so there is an example with a charset.